lib/ZnapZend/ZFS.pm

```diff
@@ -335,7 +335,7 @@ sub sendRecvSnapshots {
 
     my @cmd;
     if ($lastCommon){
-        @cmd = ([@{$self->priv}, 'zfs', 'send', @sendOpt, '-I', $lastCommon, $lastSnapshot]);
+        @cmd = ([@{$self->priv}, 'zfs', 'send', @sendOpt, '-i', $lastCommon, $lastSnapshot]);
```

Comment:
if znapzend has been creating local snapshots for some time, without being able to send them to the destination (dest was down maybe) then, calling -I will ensure that all the snapshots that did NOT go out to dest will be sent ... 

It is true, that if you create your own snapshots by hand, they will go out too ... so if you want to make sure that only znapzend made znapshots stay on dest, you would have to implement an option to clean unwanted snapshots from the destination ...

Reply:
Ok, which leads us back to https://github.com/oetiker/znapzend/pull/230. I do see your point, which suggests that an option like the one (or similar to the one) proposed in https://github.com/oetiker/znapzend/pull/230 should be added. I remain convinced, however, that the default should be `-i`. In my case, `-I` has proven to be a problem and indeed required me to have a periodic script that destroyed unwanted snapshots. I fail to see why that should be a default (and expected) behavior.

Reply:
the default is -I because in regards to backup it is better to err on the side of caution ... as I said, I think the simplest solution would be to have the option of removing any snapshots from destination that do not comply with the naming scheme chosen in your znapzend setup.